pkg/tinydtls/Makefile.include

```diff
@@ -11,6 +11,11 @@ INCLUDES += -I$(PKG_BUILDDIR)
 # Mandatory for tinyDTLS
 CFLAGS += -DDTLSv12 -DWITH_SHA256
 
+PSEUDOMODULES += tinydtls_debug
+ifeq (,$(filter tinydtls_debug, $(USEMODULE)))
+  CFLAGS += -DNDEBUG=1
```

Comment:
Won't this pollute the name space of all compilation units with an `NDEBUG` symbol? Maybe call it `TINYDTLS_NDEBUG` instead, to avoid name clashes?

Reply:
The symbol is already used by tinyDTLS.
Isn't the `Makefile.include` restricted to the module itself?
Or was it `Makefile` - I always mix that up.